docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md

```diff
@@ -78,6 +78,8 @@ python3 mo.py --input_model wsj_dnn5b_smbr.nnet --counts wsj_dnn5b_smbr.counts -
 	\f$|C|\f$ - number of elements in the counts array;
   * The normalized counts are subtracted from biases of the last or next to last layer (if last layer is SoftMax).
 
+  > **NOTE:** Model Optimizer will show warning if model contains counts values inside model. But counts values will not applyed if `--counts` option is not used.
```

Comment:
will not applyed -> will not be applied.
I have the same comment as above. It looks that we need to use `--count` option if count values are inside but it is not clear what `COUNTS` should equal.